src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -218,10 +218,23 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                 ->fixXmlConfig('delete_cookie')
                 ->children()
                     ->arrayNode('delete_cookies')
+                        ->normalizeKeys(false)
```

Comment:
I should remove this line too, right? We must keep normalizing keys until Symfony 5.0.

Reply:
correct!

Reply:
or you can keep it, and normalize in the closure - that might be easier for 5.0

Reply:
Much better! Done. Thanks.